problemsets-mat223/modules/module5.tex

```diff
@@ -155,15 +192,31 @@
 \begin{example}
 	Compute the projection of $\vec a=\mat{3\\7}$ onto $\ell=\Span\Set*{\mat{1\\-4}}$.
 
-	XXX Finish
+	By the theorem above, let $\vec b = \mat{1\\-4}$, we have
+	\[
+	    \Proj_\ell \vec a = \Vcomp_{\vec b} \vec a = \left(\frac{\vec d\cdot \vec a}{\vec b\cdot \vec b}\right)\vec b = -\frac{3-28}{1+16}\mat{1\\-4}=-\frac{25}{17}\mat{1\\-4}.
+	\]
 \end{example}
 
 It's worth noting, however, that vector components are equal to projections \emph{only in the case when you're
 projecting onto a span}. In general, projections and vector components are unrelated. 
 
 \begin{example}
 	Let $\vec a=\mat{3\\7}$ , $\vec b=\mat{1\\-4}$, and let $\ell$ be the line given in vector form by
-	$\vec x=t\vec b+\vec a$. Show that $\Proj_\ell\vec a\neq \Vcomp_{\vec v}\vec a$.
+	$\vec x=t\vec b+\vec a$. Show that $\Proj_\ell\vec a\neq \Vcomp_{\vec b}\vec a$.
 
-	XXX Finish
+	By definition, $\Proj_{\ell}\vec a$ is the closest point in $\ell$ to $\vec a$. Since the distance is given by
+	\[
+	    \norm*{(t\vec b+\vec a)-\vec a}=t\norm*{\vec b}=\sqrt{17}t,
+	\]
+	where $t=0$ minimizes the value of the distance. Therefore, we obtain
+	\[
+	    \Proj_{\ell}\vec b = \vec 0 + \vec a = \mat{3\\7}.
```

Comment:
You messed up your computation. [3, 7] isn't on `\ell`

Reply:
Prof, the question you give here is:
`\ell`: x = = t`\vec b` + `\vec a` = t(1, 4) + (3, 7).
You make the trick `\ell` is `\span{ \vec b} + \vec a` to trick students. ^-^
I should be confident to my calculation since I am you students hahaha.

Reply:
Haha, excellent. I had `a` and `b` swapped in my head ;-)